compiled_starters/javascript/.gitignore

```diff
@@ -1 +1 @@
-*.db
```

Comment:
@ryan-gang we probably still want the `*.db` here (when users download the sample DBs, they'll end up in their git history otherwise, and these are 100+ MBs)